mesa_geo/geospace.py

```diff
@@ -48,6 +46,32 @@ def __init__(self, crs="epsg:3857"):
         self.idx = index.Index()
         self.idx.agents = {}
 
+    @property
+    def crs(self):
+        """
+        Return the coordinate reference system of the GeoSpace.
+        """
+        return self._crs
+
+    @property
+    def transformer(self):
+        """
+        Return the pyproj.Transformer that transforms the GeoSpace into
+        epsg:4326. Mainly used for GeoJSON serialization.
+        """
+        return self._transformer
+
+    @crs.setter
+    def crs(self, crs):
```

Comment:
Under which situation is the `crs` ever updated after the initialization? Just wondering if it could be simpler if both `crs` and `transformer` are read-only -- no setter. And you can instead do `self._crs = ...` in `__init__`. By simpler, I meant that the code is easier to understand.

Reply:
This is a good point. Thank you! I can't think of any reason to change the `crs` of `GeoSpace` while the model is running. (We might be wrong but let's see.)

Have updated the code accordingly.